UltraStar Play/Assets/Scenes/SongSelect/SongSelectSceneControl.cs

```diff
@@ -84,6 +87,9 @@ public static SongSelectSceneControl Instance
     [Inject(UxmlName = R.UxmlNames.songIndexButton)]
     private Button songIndexButton;
 
+    [Inject(UxmlName = R.UxmlNames.buttonJoker)]
+    private Button buttonJoker;
```

Comment:
- It is a button, should be called `jokerButton`. In general, most defining stuff at the end of the name.